packages/react/src/contexts/ClerkProvider.tsx

```diff
@@ -26,7 +26,7 @@ function ClerkProviderBase(props: ClerkProviderProps): JSX.Element {
       errorThrower.throwMissingFrontendApiOrPublishableKeyError();
     } else if (publishableKey && !isPublishableKey(publishableKey)) {
       errorThrower.throwInvalidPublishableKeyError({ key: publishableKey });
-    } else if (frontendApi && !isLegacyFrontendApiKey(frontendApi)) {
+    } else if (!publishableKey && frontendApi && !isLegacyFrontendApiKey(frontendApi)) {
```

Comment:
❓ At which point do we populate `frontendApi` here?

Reply:
Yes same, i am curious why the frontendApi will be populated here if someone pass a publishableKey

Reply:
This comes from backend actually. It has to do with SSR.

Here it is:
https://github.com/clerkinc/javascript/blob/20b7f0934a99923961e930a3e3519ace70cd9457/packages/backend/src/tokens/request.ts#L63

Reply:
The error we get comes from the server

Reply:
That makes sense, thanks